server/server.js

```diff
@@ -14,14 +14,14 @@ app.use("/directives", express.static(path.join(__dirname, "../client/common/dir
 app.use("/services", express.static(path.join(__dirname, "../client/common/services")));
 app.use("/jquery", express.static(path.join(__dirname, "../node_modules/jquery/dist")));
 app.use("/bootstrap", express.static(path.join(__dirname, "../node_modules/bootstrap/dist")));
-app.use("/font-awesome", express.static(__dirname + "/../node_modules/font-awesome"));
 app.use("/css", express.static(path.join(__dirname, "../client/css")));
 app.use(express.static(path.join(__dirname, "../client/views")));
 app.use(bodyParser.json());
 
 app.use("/", routes);
 
-app.listen(3000);
-console.log("Server running on port 3000...");
+var port = process.env.PORT || 8080;
```

Comment:
This is a small change I added because I just ran into this problem myself. If you were to host this, you need to let the process try to map the port itself if it wants to, and if not, use 8080.